tools/check_srcrevs.py

```diff
@@ -10,18 +27,18 @@ def findRev(file):
 
 def updateOeVersion(file, newVersion):
     # Read contents from file as a single string
-    file_handle = open(file, 'r')
-    file_string = file_handle.read()
-    file_handle.close()
+    fd = open(file, 'r')
+    file_content = fd.read()
+    fd.close()
 
     # Use RE package to allow for replacement (also allowing for (multiline) REGEX)
-    file_string = (re.sub("[a-z0-9]{40}", newVersion, file_string))
+    file_content = (re.sub("[a-z0-9]{40}", newVersion, file_content))
 
     # Write contents to file.
     # Using mode 'w' truncates the file.
```

Comment:
Change the comments at 29, 34 and 37 so they are not identical and therefore a direct copy.  e.g.
30: # Get file contents, multiple lines returned as one string
34: # Re package will handle multiline input string in substitution
37/38: # Write contents and truncate file.